pages/design/rule-template.md

```diff
@@ -4,7 +4,7 @@ title: Rule Template
 
 The rule template contains a plain language description of the rule, some background information, and a list of all assumptions. The test procedure is defined by the selector, a number of steps and a description of the possible outcomes.
 
-Use the [empty atomic rule template](atomic-template-empty.html) to create new rule. When creating a new rule, first read [rule design](rule-design.html).
+Use the [empty atomic rule template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/atomic-template-empty.md) to create new rule. When creating a new rule, first read [rule design](/pages/design/rule-design/).
```

Comment:
The `pages/design/rule-design` link seems broken (at least when I try to view this file and click on this link, I get a 404) and might need to be `/pages/design/rule-design.md` instead. (but I’m not sure how much magic is happening in the back…)

Reply:
@Jym77 - this is what this will get resolved to https://act-rules.github.io/pages/design/rule-design/

All markdown content is converted to html pages & this is one such page.

Reply:
OK. I suspected some of it, but hadn’t had time to look into the code to figure all that out.